tests/types/AntiCheat.Sentry.txt

```diff
@@ -0,0 +1,4 @@
+sentry
```

Comment:
I haven't looked at how many games match, but sentry is also a error reporting tool, so don't rely just on the base name.

Also pretty sure it makes `sentryabc` here.